client/src/app/pages/FAQ/FAQ.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="container about-page">
+
+  <h1>Frequently asked questions</h1>
+
+  <div class="row">
+    <div class="col-full">
+
+    <p>
+      Questions that we are asked quite often, and some answers.
+    </p>
+
+      <h1>FAQS</h1>
+
+        <div>
+          
+          <h2>CubingZA Questions</h2>
+
+            <h3><b>1.</b> What can I do to help CubingZA?</h3> <p>If you'd like to help out you can read through <a href="/volunteer">this page</a> to learn more about how you can help.</p> 
+
+          <h2>General Questions</h2>
+
+            <h3><b>2.</b>How do I stay in touch with other cubers?</h3> <p>Join the CubingZA Whatsapp community by clicking on <a href="https://chat.whatsapp.com/IIDXPTFGhJw7u3gUl4Nf4I">this link!</a></p> 
+
+          <h2>Competition Questions</h2>
+
+            <h3><b>3.</b> Do I need to be at both days if I am only competing on one day?</h3>
+            No, you only have to be there on the day that you are competing. Make sure to check out the schedule tab to see when you compete. We suggest arriving at the competition at least 30 minutes prior to your event in case we run ahead of schedule.
+
+            <h3><b>4.</b>Where can I view my results on the day of the competition?</h3>
+            On the competition days all results will be uploaded to WCA Live. Just search for the competition name under the "upcoming" tab. After the competition has ended, it will take a few days for the official results to be uploaded to the WCA main site.
```

Comment:
Please make the text "WCA Live" link to the WCA Live website.

Reply:
Done